pymobiledevice3/services/crash_reports.py

```diff
@@ -150,18 +151,25 @@ def _wait_for_sysdiagnose_to_finish(self) -> None:
 
     def _get_new_sysdiagnose_filename(self) -> str:
         sysdiagnose_filename = None
+        excluded_temp_files = []
 
         while sysdiagnose_filename is None:
             try:
                 for filename in self.afc.listdir(SYSDIAGNOSE_DIR):
                     # search for an IN_PROGRESS archive
-                    if 'IN_PROGRESS_' in filename:
+                    if filename not in excluded_temp_files and 'IN_PROGRESS_' in filename:
                         for ext in self.IN_PROGRESS_SYSDIAGNOSE_EXTENSIONS:
                             if filename.endswith(ext):
-                                sysdiagnose_filename = filename.rsplit(ext)[0]
-                                sysdiagnose_filename = sysdiagnose_filename.replace('IN_PROGRESS_', '')
-                                sysdiagnose_filename = f'{sysdiagnose_filename}.tar.gz'
-                                return posixpath.join(SYSDIAGNOSE_DIR,  sysdiagnose_filename)
+                                delta = datetime.now() - self.afc.stat(posixpath.join(SYSDIAGNOSE_DIR,filename))['st_mtime']
```

Comment:
use the device time since it may be different then the workstation

Reply:
Done